doc/setting_up_sidewalk_environment/setting_up_hardware.rst

```diff
@@ -88,6 +88,73 @@ Connect the Semtech shield to the Arduino-compatible headers on the development
     +---------------+-------------+
     | NRESET        | P0.04       |
     +---------------+-------------+
+   
+   .. tab:: nRF54l15pdk
+
+      The nrf54l15pdk does not have Arduino compatible header.
+      The connection layout may differ depending on the revision of the pdk used.
+      Those GPIO assignments are suggested by Nordic team, but if it is necessary to change the pinout, please see the SoC documentation, as some functions are limited to dedicated pins. 
+
+      The nrf54l15pdk operates on lower voltage level (1.8V). Semtech Shield is compatible with this voltage, and it can be safely provided to the pin marked as 3.3V.
+      Due to the lower voltage the Semtech radio can not achieve the full transmit power, and it will plato at 15dBm gain, even when bigger value is configured.
```

Comment:
```suggestion
      Due to the lower voltage, the Semtech radio cannot achieve the full transmit power, and it will reach a plateau at 15 dBm gain even when a higher value is configured.
```